SonarScanner.Shim/PropertiesFileGenerator.cs

```diff
@@ -226,9 +226,9 @@ private static ProjectData GetClosestProjectOrDefault(string filePath, IEnumerab
 
                 // Ensure that folder path always ends with backslash so a directory like 'c:\aaa\bbb' doesn't match
                 // a file like 'c:\aaa\bbb.cs' nor 'c:\aaa\bbbxxxx\myfile.cs'
-                if (!projectPath.EndsWith(@"\"))
+                if (!projectPath.EndsWith(Path.DirectorySeparatorChar.ToString()))
```

Comment:
EndsWith requires string as an argument, hence ToString()

Reply:
stupid me...

Reply:
Do we need to consider _Path.AltDirectorySeparatorChar_ as well?
Perhaps not, because _Project.GetProjectDirectory()_ has already called _Path.GetFullPath()_, and that might have converted the directory separators to the "normal" form for the platform.